### DISCUSSION TITLE
[feature request] audio tagging with Whipser-AT

### DISCUSSION THREAD

**Discussion by phineas-pta** [SYSTRAN/faster-whisper#352] [Ideas]
so somebody manage to integrate audio tagging with whisper:
- announcement: openai/whisper#1504
- source code: https://github.com/YuanGongND/whisper-at

it seems they dont modify original whisper architecture (load directly openai checkpoints), the audio tagging feature is somehow hooked into it and load their pretrained checkpoint

it's unlikely the author could implement faster-whisper: YuanGongND/whisper-at#1

**Comment by Ar770** [comment]
Any plans to add this feature?

**Comment by kuroderuta** [comment]
Yeah, this would be really useful.
